# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/nsFrameLoader.cpp
# Commit: 21985ab71b5b
# Full Hash: 21985ab71b5b6f052ec26b4341db0eb47e589f32
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2019-05-23 15:55:11
# Regressor Bug: 1525720
# File Overlap Count: 1
# Description:
#   Bug 1525720, part 6 - Move BrowserBridgeChild creation to ContentChild. r=farre
#   
#   This makes it symmetrical to how BrowserParent is created by ContentParent.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31436
# ==============================================================================

diff -r 420f10c25da1 -r 21985ab71b5b dom/base/nsFrameLoader.cpp
--- a/dom/base/nsFrameLoader.cpp	Wed May 15 12:33:42 2019 -0500
+++ b/dom/base/nsFrameLoader.cpp	Wed Apr 24 23:52:47 2019 -0500
@@ -2715,7 +2715,7 @@
   if (XRE_IsContentProcess()) {
     mBrowsingContext->SetEmbedderElement(mOwnerContent);
 
-    mBrowserBridgeChild = BrowserBridgeChild::Create(
+    mBrowserBridgeChild = ContentChild::CreateBrowser(
         this, context, NS_LITERAL_STRING(DEFAULT_REMOTE_TYPE),
         mBrowsingContext);
     return !!mBrowserBridgeChild;